src/components/sbb-form-field/sbb-form-field.stories.tsx

```diff
@@ -143,21 +143,21 @@ const TemplateInputWithButton = ({ disabled, readonly, active, ...args }): JSX.E
     {TemplateBasicInput({ ...args, disabled, readonly })}
     <sbb-button
       slot="suffix"
-      icon-name="cross-small"
+      icon-name="pie-small"
       disabled={disabled || readonly}
-      aria-label="clear input"
+      aria-label="Input button"
       data-active={active}
-      onClick={() => {
-        const input = document.querySelector('input');
-        if (input) {
-          input.value = '';
-          input.focus();
-        }
-      }}
     />
   </sbb-form-field>
 );
 
+const TemplateInputWithClearButton = ({ disabled, readonly, ...args }): JSX.Element => (
```

Comment:
active argument needs special handling, see TemplateInputWithButton

Reply:
Yep, sorry I missed that